EasyPost/BetaFeatures/Parameters/Addresses/Create.cs

```diff
@@ -4,21 +4,28 @@
 namespace EasyPost.BetaFeatures.Parameters.Addresses
 {
     /// <summary>
-    ///     Parameters for <see cref="EasyPost.Services.AddressService.Create(Create)"/> API calls.
+    ///     <a href="https://www.easypost.com/docs/api#create-and-verify-addresses">Parameters</a> for <see cref="EasyPost.Services.AddressService.Create(Create, System.Threading.CancellationToken)"/> API calls.
     /// </summary>
     [ExcludeFromCodeCoverage]
     public sealed class Create : BaseParameters, IAddressParameter
     {
         #region Request Parameters
 
-        // ID can't be provided when creating an address, but can be provided when using an address in a non-address creation request.
+        /// <summary>
+        ///     ID of the <see cref="Models.API.Address"/> to reference in this request.
+        ///     ID is not used when calling <see cref="Services.AddressService.Create(Create, System.Threading.CancellationToken)"/>,
+        ///     but is used when using this parameter set as a nested parameter set in other API calls.
+        /// </summary>
         [NestedRequestParameter(typeof(Shipments.Create), Necessity.Optional, "id")]
         [NestedRequestParameter(typeof(Insurance.Create), Necessity.Optional, "id")]
         [NestedRequestParameter(typeof(Orders.Create), Necessity.Optional, "id")]
         [NestedRequestParameter(typeof(Pickups.Create), Necessity.Optional, "id")]
         [NestedRequestParameter(typeof(Beta.Rates.Retrieve), Necessity.Optional, "id")]
         public string? Id { get; set; }
 
+        /// <summary>
+        ///     Carrier facility for the new <see cref="Models.API.Address"/>.
+        /// </summary>
```

Comment:
thought: as suggested initially, I feel that documenting all the properties is taking documentation to the extreme. Most should be self explanatory, would they even pop up in an IDE?

Documenting functions and classes makes sense because they have logic associated that may need additional explanations where properties should be exactly the name given to them.

Thoughts?

Reply:
Yes, they would pop up in an IDE, definitely.

As a user, I might not know what some of these things are, or when/if I should set them to specific values.